tests/utils/test_tensors.py

```diff
@@ -95,12 +95,24 @@ def test_contiguous():
     assert type(expandable) != ExpandableTensor
 
 
+def get_paged_tensor():
+    batches = 5
+    # default page size is 16. using 15 x 7 ensures un-even paging, to ensure
+    # copy works correctly.
+    seq_len = 15 * 7
+    emb_dim = 4
+    t = torch.arange(batches * seq_len * emb_dim).view(batches, seq_len, emb_dim)
+    pt = PagedTensor.from_tensor(t, [-1])
+    return pt
+
+
 def test_paged():
+    # simplest test, the number of items is evenly divisible by page size
     batches = 5
     seq_len = 16 * 7
     emb_dim = 4
     t = torch.arange(batches * seq_len * emb_dim).view(batches, seq_len, emb_dim)
-    pt = PagedTensor(t, [-1])
+    pt = PagedTensor.from_tensor(t, [-1])
```

Comment:
could you use `get_paged_tensor` here?

Reply:
IMO 'get' implies getting an attribute from an instance of the class. This is a factory method (class method)